src/core/Icon/Icon.tsx

```diff
@@ -11,13 +11,18 @@ import {
   IconProps as CompIconProps,
 } from '../../components/Icon/Icon';
 import { Omit } from '../../utils/typescript';
-import { SuomifiIcon, SuomifiIconInterface, IconKeys } from 'suomifi-icons';
-export { IconKeys } from 'suomifi-icons';
+import {
+  SuomifiIcon,
+  SuomifiIconInterface,
+  IconKeys,
+  StaticIconKeys,
+} from 'suomifi-icons';
+export { IconKeys, StaticIconKeys } from 'suomifi-icons';
 import { logger } from '../../utils/logger';
 
 export interface IconProps extends Omit<CompIconProps, 'src'> {
   /** Icon-name from suomifi-icons */
-  icon?: IconKeys;
+  icon?: IconKeys | StaticIconKeys;
```

Comment:
Ah would have been nice to export only one interface combining these byt yeah lets go with this